src/js/background.js

```diff
@@ -766,6 +789,51 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  showCookies: function(fqdn) {
+    let domain = window.getBaseDomain(fqdn);
+    chrome.cookies.getAll({domain}, cookies => {
+      console.log('Cookies for domain: ' + domain);
+      cookies.forEach(cookie => {
+        console.log('\t -------------');
+        console.log('\t cookie name: ' + cookie.name);
+        console.log('\t cookie domain: ' + cookie.domain);
+      });
+    });
+  },
+
+  debugSite: function() {
+    let self = this;
+    chrome.tabs.query({'active': true}, tabs => {
+      tabs.forEach(tab => {
+        let action_map = self.storage.getBadgerStorageObject('action_map').getItemClones(),
+          snitch_map = self.storage.getBadgerStorageObject('snitch_map').getItemClones(),
+          seen = new Set(),
+          out = {
+            info: {
+              origins: [],
+              action_maps: {},
+              snitch_maps: {},
+              browser: window.navigator.userAgent,
+              version: chrome.runtime.getManifest().version,
+              fqdn: window.extractHostFromURL(tab.url),
+            }
+          };
+        console.log('------- Debug infor for tab with url: ' + tab.url + ' -------');
+        self.getAllOriginsForTab(tab.id).forEach(origin => {
+          let base = window.getBaseDomain(origin);
+          out.info.origins.push(origin);
+          for (let am_fqdn in action_map) {
```

Comment:
Could we document why we iterate over `action_map` and make base domain comparisons somewhere around here?